utils/_features.py

```diff
@@ -0,0 +1,166 @@
+import pytest
+import requests
+
+
+def _feature(feature_id: int):
+    def decorate(test_object):
+        pytest.mark.features(feature_id=feature_id)(test_object)
+        return test_object
+
+    return staticmethod(decorate)
+
+
+class features:
```

Comment:
Why not include all feature IDs so that tests can be added for the other features?